workers/tests/integrationtests/test_geo_interface.py

```diff
@@ -0,0 +1,29 @@
+import asyncio
+import unittest
+
+import geojson
+
+from ohsome_quality_analyst.geodatabase import client as db_client
+from ohsome_quality_analyst.indicators.ghs_pop_comparison_buildings.indicator import (
+    GhsPopComparisonBuildings,
+)
+from ohsome_quality_analyst.utils.helper import datetime_to_isostring_timestamp
+
+
+class TestGeoInterface(unittest.TestCase):
+    def test(self):
+        feature = asyncio.run(
+            db_client.get_region_from_db(feature_id=3, fid_field="ogc_fid")
+        )
+        indicator = GhsPopComparisonBuildings(
+            feature=feature, layer_name="building_count"
+        )
+        print(indicator.feature)
+        print(indicator.__geo_interface__)
```

Comment:
Traceback (most recent call last):
  File "/home/matthias/work/oqt/workers/tests/integrationtests/test_geo_interface.py", line 22, in test
    print(indicator.__geo_interface__)
  File "/home/matthias/work/oqt/workers/ohsome_quality_analyst/base/indicator.py", line 108, in __geo_interface__
    "geometry": self.feature.geometry,
AttributeError: 'GhsPopComparisonBuildings' object has no attribute 'feature'